cmd/csaf_uploader/main.go

```diff
@@ -435,6 +435,8 @@ func main() {
 	}
 
 	for _, arg := range args {
-		check(p.process(arg))
+		if err := p.process(arg); err != nil {
```

Comment:
This changes the semantics a bit, because `check()` will use os.exit(0) in some cases, the new code doesn't. @s-l-teichmann is this intentional?

Reply:
`check`  only does `os.exit(0)` if the command line help was written. This case does not apply here. All errors come from the processing.